extended/src/test/java/apoc/ml/OpenAIOpenLMIT.java

```diff
@@ -31,12 +34,169 @@ public class OpenAIOpenLMIT {
     @Rule
     public DbmsRule db = new ImpermanentDbmsRule();
 
-
     @Before
     public void setUp() throws Exception {
         TestUtil.registerProcedure(db, OpenAI.class);
+
     }
 
+    @Test
+    public void completionWithAnthropic() {
+        String anthropicApiKey = System.getenv("ANTHROPIC_API_KEY");
+        Assume.assumeNotNull("No ANTHROPIC_API_KEY environment configured", anthropicApiKey);
+
+        Map<String, Object> conf = Map.of(
+                ENDPOINT_CONF_KEY, "https://api.anthropic.com/v1/messages",
+                API_TYPE_CONF_KEY, OpenAIRequestHandler.Type.ANTHROPIC.name(),
+                MAX_TOKENS, 1024
+        );
+        testCall(db, COMPLETION_QUERY_EXTENDED,
+                Map.of("conf", conf, "apiKey", anthropicApiKey),
+                (row) -> {
+                    var result = (Map<String,Object>) row.get("value");
+                    var contentList = (List<Map<String, Object>>) result.get("content");
+                    Map<String, Object> content = contentList.get(0);
+                    String generatedText = (String) content.get("text");
+                    assertTrue(generatedText.toLowerCase().contains("blue"),
+                            "Actual generatedText is " + generatedText);
+                });
+    }
+
+    @Test
+    public void completionWithAnthropicNonDefaultModel() {
+        String anthropicApiKey = System.getenv("ANTHROPIC_API_KEY");
+        Assume.assumeNotNull("No ANTHROPIC_API_KEY environment configured", anthropicApiKey);
+
+        String modelId = "claude-3-haiku-20240307";
+        Map<String, Object> conf = Map.of(
+                ENDPOINT_CONF_KEY, "https://api.anthropic.com/v1/messages",
```

Comment:
In effetti visto che abbiamo messo l'endpoint di default in `OpenAIRequestHandler.java`,
forse se nel `case ANTHROPIC -> {` 

metti `configuration.put(PATH_CONF_KEY, "messages");` invece di `configuration.put(PATH_CONF_KEY, "");`,
dovrebbe funzionare anche senza mettere `ENDPOINT_CONF_KEY, "https://api.anthropic.com/v1/messages",` qui.

Nel caso vada, toglilo anche dagli altri e dalla documentazione

Reply:
si possiamo fare a meno di endpoint